.github/workflows/deploy.yml

```diff
@@ -1,7 +1,18 @@
 name: Trigger Deployment
 
 # see: https://help.github.com/en/actions/reference/events-that-trigger-workflows
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      root_path:
+        description: "Root path to deploy the Muse App"
+        default: "/muse"
```

Comment:
isn't `root_path` static?

Reply:
if someone wishes to deploy to some other root path

Reply:
It's a param to the `LightningApp`

Reply:
but they would have to update the app name (Stable Diffusion) as well, right?
Also, `someone` here means "folks having access to the Prod app tokens" right?

Reply:
yup, reverting the change, we could discuss it in another PR